index.html

```diff
@@ -33,27 +33,27 @@ <h2 id="title2">I'm a software developer</h2>
             <ul class="social-media flex">
                 <li>
                     <a href="!">
-                        <img src="images/twitter.svg">
+                        <img src="images/twitter.svg" alt="Twitter icon to be in contact">
                     </a>
                 </li>
                 <li>
                     <a href="!">
-                        <img src="images/Linkedin icon.svg">
+                        <img src="images/Linkedin icon.svg" alt="Linkedin icon to be in contact">
                     </a>
                 </li>
                 <li>
                     <a href="!">
-                        <img src="images/medium.svg">
+                        <img src="images/medium.svg" alt="Medium icon to be in contact">
                     </a>
                 </li>
                 <li>
                     <a href="!">
-                        <img src="images/github.svg">
+                        <img src="images/github.svg" alt="GitHub icon to be in contact">
                     </a>
                 </li>
                 <li>
                     <a href="!">
-                        <img src="images/angellist.svg">
+                        <img src="images/angellist.svg" alt="Angellist icon to be in contact">
                     </a>
                 </li>
```

Comment:
-[Optional] Great job for improving the previous reviewer selection 👍 , I think it would be better if you minimize the gap between social icons to stick to the given design and have a better UI 👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/235227328-b4c52096-a7d8-4d7f-aa5f-84ae7ae8650c.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/235227434-bde67ef2-32af-4132-855a-b8dc44689018.png" width="200">|


😎️

🤝️